doc/service-agent-api.md

```diff
@@ -154,14 +154,14 @@ Parameters:
 - (optional) Revocation Index
 - (optional) Claims
 
-> **Note:** When using revocation parameters (`revocationDefinitionId` and `revocationRegistryIndex`), it is essential to ensure the `id` was preserved from the time it was generated with the credential. The `revocationRegistryIndex` serves as a reference to the specific credential in the revocation registry.  
+> **Note:** When using revocation parameters (`revocationRegistryDefinitionId` and `revocationRegistryIndex`), it is essential to ensure the `id` was preserved from the time it was generated with the credential. The `revocationRegistryIndex` serves as a reference to the specific credential in the revocation registry.  
```

Comment:
> it is essential to ensure the `id` was preserved

What do you mean by 'id'? Identifier? Index? Both? Try to be clear on these terms because it will be very confusing to readers.

Reply:
fixed